walkthroughs/http-headers-and-priority/k8s/deploy.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -e
+
+if [ -z "${AWS_ACCOUNT_ID}" ]; then
+    echo "AWS_ACCOUNT_ID must be set."
+    exit 1
+fi
+
+if [ -z "${AWS_DEFAULT_REGION}" ]; then
+    echo "AWS_DEFAULT_REGION must be set."
+    exit 1
+fi
+
+if [ -z "${MESH_NAME}" ]; then
+    echo "EKS_CLUSTER must be set."
+    exit 1
+fi
+
+MESH_NAME=${MESH_NAME:-"headerpriority"}
+APP_NAMESPACE=${APP_NAMESPACE:-"headerpriority"}
+COLOR_IMAGE_NAME="${APP_NAMESPACE}-colorapp"
+COLOR_IMAGE="${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com/${COLOR_IMAGE_NAME}"
+FRONT_IMAGE_NAME="${APP_NAMESPACE}-feapp"
+FRONT_IMAGE="${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com/${FRONT_IMAGE_NAME}"
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
+OUT_DIR="${DIR}/_output/"
+mkdir -p ${OUT_DIR}
+
+for f in mesh app; do
+    eval "cat <<EOF
+$(<${DIR}/${f}.yaml.template)
+EOF" >${OUT_DIR}/${f}.yaml
```

Comment:
You might use the `envsubst` program instead of the above use of `eval`:

```bash
for f in mesh app; do
    envsubst <$DIR/$f.yaml.template >$DIR/$f.yaml
done
```

https://www.gnu.org/software/gettext/manual/html_node/envsubst-Invocation.html

You'd want to `export` the vars used in the template, though.

Reply:
Don't I need to install gettext for this to work?

Reply:
Pretty sure `envsubst` is a standard utility on Linux at this point, but not a huge deal :)

Reply:
Not on my mac :(. Will leave it for now. Thanks for pointer.